test/sqlite/test-sqlite.mjs

```diff
@@ -0,0 +1,85 @@
+import '../common/index.mjs';
+
```

Comment:
Can you rename this file please. Something like `test-sqlite-extensions.mjs` should work.